src/service/triple-store-service.js

```diff
@@ -215,13 +216,23 @@ class TripleStoreService {
         }
 
         // TODO: add new metadata triples and move to function insertMetadataTriples
-        const metadataTriples = publicKnowledgeAssetsUALs
+        let metadataTriples = publicKnowledgeAssetsUALs
             .map(
                 (publicKnowledgeAssetUAL) =>
                     `<${publicKnowledgeAssetUAL}> <http://schema.org/states> "${publicKnowledgeAssetUAL}:0" .`,
             )
             .join('\n');
-        totalNumberOfTriplesInserted += publicKnowledgeAssetsUALs.length; // one metadata triple for each public KA
+
+        metadataTriples +=
+            `\n<${knowledgeCollectionUAL}> <https://ontology.origintrail.io/dkg/1.0/#publishedBy> <did:dkg:publisherKey:${metadata.publisherKey}> .` +
```

Comment:
Instead of 
`<did:dkg:publisherKey:${metadata.publisherKey}>`

I would 

<did:dkg:publisherKey/${metadata.publisherKey}>

To stay consistent with UAL structure